### ISSUE TITLE
A11y_AzureToolsStorageExplorer_RenameBlobContainer_ScreenReader: Voiceover does not read the alert message while Voiceover focus goes to the edit box

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/AzureStorageExplorer#1958]
[Check out Accessibility [Insights!]-](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;Voiceover
**Environment Details:**
Desktop App: MAC OS Mojave
Version:10.14.6(18G29g)
**Steps to Reproduce:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once created, navigate to the created blob container and right click on it.
6. Select "Rename" and hit enter.
7. Open Voiceover Check whether the Voiceover reads the dialog name with the edit field name.

**Actual:**
While navigating through keyboard, Voiceover does not read the alert message while Voiceover focus goes to the edit box.
Same issue is observed throughout the application for all the alert message while Voiceover focus goes to the edit box.
**Expected:**
Voiceover should read the alert message after read the Please Specify the new name.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**User Impact:**
Screen reader user unable to know that which dialog is opened.
**MAS Reference:**
MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**Attachment for Reference**
[1958_A11y_AzureToolsStorageExplorer_RenameBlobContainer_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3720646/1958_A11y_AzureToolsStorageExplorer_RenameBlobContainer_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
Fix checked into master for 1.12

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by nirmalsahoo** [comment]
@MRayermannMSFT  Verified the issue on version 1.12.0 and MAC 10.15.3.The issue is no longer reproduced.Hence close this bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;Mac;
